packages/web-components/stencil.config.ts

```diff
@@ -24,10 +25,10 @@ export const config: Config = {
   // https://stenciljs.com/docs/config-extras
   buildEs5: 'prod',
   extras: {
-    cssVarsShim: true,
-    dynamicImportShim: true,
-    shadowDomShim: true,
-    safari10: true,
+    __deprecated__cssVarsShim: true,
```

Comment:
We should add a ticket to re-evaluate these properties now that they are deprecated. I believe most of them are to support older browser versions which we no longer support.

Reply:
That's a good idea. Can we link to that ticket here?